sentry-core/src/main/java/io/sentry/core/transport/HttpTransport.java

```diff
@@ -162,6 +163,11 @@ public HttpTransport(
         final Writer writer = new BufferedWriter(new OutputStreamWriter(outputStream, UTF_8))) {
       serializer.serialize(event, writer);
 
+      // TODO: remove this, just logging
+      StringWriter wrt = new StringWriter();
+      serializer.serialize(event, wrt);
+      options.getLogger().log(INFO, "event: %s", wrt.toString());
+
```

Comment:
only for logging and testing...

Reply:
What about a `logger.isEnabled(LogLevel.debug)) { ...` ?
This way we only pay for the serialization if logging is enabled for that log level (debug).

Reply:
I'd not log the whole payload, when this gets out of a Draft, I'll remove this lines, see the TODO above